experimental/packages/opentelemetry-instrumentation-http/src/utils.ts

```diff
@@ -110,8 +110,34 @@ export const getAbsoluteUrl = (
   ) {
     host += `:${port}`;
   }
-
-  return `${protocol}//${host}${path}`;
+  // Redact sensitive query parameters
+  if (path.includes('?')) {
+    const [pathname, query] = path.split('?', 2);
+    const redactedParams = query.split('&').map(param => {
+      // List of sensitive parameter names to redact
+      // Add more as needed
+      const sensitiveParams = [
+        'sig',
+        'Signature',
+        'AWSAccessKeyId',
+        'X-Goog-Signature',
+      ] as const;
+      const paramParts = param.split('=', 2);
+      const paramName = paramParts[0];
+      // Check if this parameter should be redacted
+      if (
+        sensitiveParams.some(sensitive =>
```

Comment:
Wouldn't you want this to be `sensitiveParams.includes(paramName.toLowerCase())`?